packages/picker/src/Picker.ts

```diff
@@ -160,6 +160,15 @@ export class PickerBase extends SizedMixin(Focusable) {
         this.toggle();
     }
 
+    public onHelperFocus(): void {
+        // set focused to true here instead of onButtonFocus so clicks don't flash a focus outline
+        this.focused = true;
+        const button = this.shadowRoot.querySelector(
```

Comment:
This should already be available as `this.button`. Swap this out and we'll be good to go!

Reply:
Good catch!

Reply:
✅